doc/branding.md

```diff
@@ -32,24 +44,26 @@ There are 3 options for dynamic branding - header logo, title and custom CSS sty
 
 ### Header logo
```

Comment:
@petrKavulok It should be good to write that when only light / dark logo is present, the switch between the themes will not have any impact on changing the logo, or something like that.